tests/tests.py

```diff
@@ -93,6 +93,7 @@ def tearDown(self):
             print(f"Test results can be found in {self.test_results}")
         else:
             self.delete_folder(self.test_results)
+        runner.run(f"rm -f {os.path.join('examples', '*.ssi')}")
```

Comment:
maybe avoid spawning a new process and use python api here?

```python
[os.remove(file) for file in glob.glob('examples/*.ssi')]
```

Reply:
Good point! I also [created an issue](https://github.com/RNAcentral/R2DT/issues/113) to go through the code and systematically reduce the number of system calls.